### PR TITLE
chore: New roadmap entry

### PR DESCRIPTION
**Pull request by sfc-gh-asawicki**
New roadmap entry with the state of the project.

### CODE REVIEW

File: v1-preparations/LIST_OF_STABLE_RESOURCES_FOR_V1.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+We estimate the given list to be accurate, but it may be subject to small changes:
+
+* Account (in progress)
+    * [snowflake_account](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/account)
+    * [snowflake_accounts](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/data-sources/accounts)
+* Connection (in progress)
+    * snowflake_connection
+    * snowflake_connections
```
Comment (sfc-gh-jmichalak) [inline v1-preparations/LIST_OF_STABLE_RESOURCES_FOR_V1.md:8]: This and `snowflake_accounts` should be marked as data sources.

File: ROADMAP.md [hunk 2]
```diff
@@ -1,5 +1,115 @@
...
+
+* [wrap up the functional scope](#wrap-up-the-functional-scope) (not all the objects will be declared stable, more details below)
+* [prepare for the V1 release](#prepare-for-the-v1-release)
+* [prepare some basic performance benchmarks](#prepare-some-basic-performance-benchmarks) (especially, after a few major changes to the resources logic)
+* [improve/update the documentation](#improveupdate-the-documentation)
+* [run a closed early adopter program](#run-a-closed-early-adopter-program) to verify the readiness of the provider to enter a stable V1
+
+If there won't be any major obstacles or critical issues we aim to release V1 on **December 9th**. To better understand its scope, please check the ["what is V1?"](#what-is-v1) section.
+
+#### reducing the feature gap
```
Comment (sfc-gh-jmichalak) [inline ROADMAP.md:24]: Nit: Capitalize

**Review by sfc-gh-jmichalak — APPROVED** [review]

File: v1-preparations/LIST_OF_STABLE_RESOURCES_FOR_V1.md (hunk 1 above)
Comment (sfc-gh-asawicki) [reply v1-preparations/LIST_OF_STABLE_RESOURCES_FOR_V1.md:8]: Done.

File: ROADMAP.md (hunk 2 above)
Comment (sfc-gh-asawicki) [reply ROADMAP.md:24]: Done.

**Review by sfc-gh-jmichalak — APPROVED** [review]

**Pull request merged**